rero_ils/modules/locations/jsonschemas/locations/location-v0.0.1.json

```diff
@@ -141,7 +141,7 @@
     },
     "send_notification": {
       "title": "Send notification",
-      "description": "Send a email notification when an item of the location has been requested.",
+      "description": "Send an e-mail notification when an item of the location has been requested.",
```

Comment:
Just checking before I change all fields to "e-mail". Source uses both and it is a better idea to use "e-mail" to avoid errors in translation I find.

Reply:
Hi,
I think you're right, *e-mail* is better. But we have to be coherent on our side too.